File: Pulsacionesdonet/ClientApp/src/app/pulsacion/models/persona.ts
export class Persona {
    Identificacion: string;
    Nombres: string;
    Apellidos: string;
    Sexo: string;
    Edad: number;
    Pulsacion: number;

    CalcularPulsacion(){
        this.Pulsacion=0;
        if(this.Sexo.toUpperCase()=='F'){
            this.Pulsacion=(220-this.Edad)/10;
        }else if(this.Sexo.toUpperCase()=='M'){
            this.Pulsacion=(210-this.Edad)/10;
        }
        
    }
}

File: Entity/Persona.cs
﻿using System;

namespace Entity
{
    public class Persona
    {
        public string Identificacion {get;set;}
        public string Nombres {get;set;}
        public string Apellidos {get;set;}
        public int Edad {get;set;}
        public string Sexo {get;set;}
        public decimal Pulsacion {get;set;}

        public void CalcularPulsaciones(){
            if(Sexo.Equals("F")|| Sexo.Equals("f")){
                Pulsacion=(220-Edad)/10;
            }else if (Sexo.Equals("M")|| Sexo.Equals("m")){
                Pulsacion=(210-Edad)/10;
            }
            
        }

    }
}